src/ACEtk/interpretation/ContinuousEnergyNeutron/Builder/EnergyDistribution/EnergyDependentWattSpectrum/Builder/src/energies.hpp

```diff
@@ -0,0 +1,16 @@
+template< typename Range,
```

Comment:
You can delete this file, it is not referenced in the builder.

Reply:
👍